src/3d-force-graph.js

```diff
@@ -137,7 +141,8 @@ export default Kapsule({
 
   stateInit: () => ({
     forceGraph: new ThreeForceGraph(),
-    renderObjs: ThreeRenderObjects()
+    renderObjs: ThreeRenderObjects(),
+    animate: () => {}
```

Comment:
Why to initialize with empty func?

Reply:
i found it a good idea to have all state variables pre-initialized. You're not using typescript / flow-type for variable types so i've added it like noop function.
If you think it is not needed, i will remove it